test-clients/src/main/java/com/hedera/services/yahcli/README.md

```diff
@@ -0,0 +1,220 @@
+# Yahcli v0.0.1
+A command line interface that is able to perform the following actions against a specified network.
+
+1. Account Operations
+    1. Get Balance.
+2. System File Operations
+    1. Download all/specific file.
+    2. Upload a System file.
+3. Fees Operations
+    1. List Basic Transaction and Query fees of all/specific service.
+
+## Getting Started
+### Setting up your environment
+
+#### Install Docker
+
+
+
+#### Create a folder structure like this.<br>
+```
+├── config.yml
+├── previewnet
+│   └── keys
+│       ├── account2.pass
+│       └── account2.pem
+```
+
+The _config.yml_ should have contents like:
+
+```
+defaultNetwork: previewnet
+networks:
+  previewnet:
+    nodes:
+      - { id: 0, account: 3, ipv4Addr: 35.231.208.148 }
+  stabletestnet:
+    nodes:
+      - { id: 0, account: 3, ipv4Addr: 34.94.106.61 }
+```
+
+You can add details of multiple networks to this config file with appropriate ipv4Addresses.
+
+_{network}/keys/_ folder should contain account{num}.pem and account{num}.pass pair for each account that you want to use for that network.
+_num_ here is the account number. 
```

Comment:
Add explanation that the `.pass` file should contain the passphrase for the `.pem` file or empty if the passphrase is ignored.